packages/cli/lib/ui/index.js

```diff
@@ -139,13 +139,114 @@ const uiBetaTag = (message, log = true) => {
   }
 };
 
+const uiInfoTag = (message, log = true) => {
+  const i18nKey = 'lib.ui';
+
+  const terminalUISupport = getTerminalUISupport();
+  const tag = i18n(`${i18nKey}.infoTag`);
+
+  const result = `${
+    terminalUISupport.color ? chalk.cyan(tag) : tag
+  } ${message}`;
+
+  if (log) {
+    logger.log(result);
+  } else {
+    return result;
+  }
+};
+
+const uiDeprecatedTag = (message, log = true) => {
+  const i18nKey = 'lib.ui';
+
+  const terminalUISupport = getTerminalUISupport();
+  const tag = i18n(`${i18nKey}.deprecatedTag`);
+
+  const result = `${
+    terminalUISupport.color ? chalk.yellow(tag) : tag
+  } ${message}`;
+
+  if (log) {
+    logger.log(result);
+  } else {
+    return result;
+  }
+};
+
+const uiErrorTag = (message, log = true) => {
+  const i18nKey = 'lib.ui';
+
+  const terminalUISupport = getTerminalUISupport();
+  const tag = i18n(`${i18nKey}.errorTag`);
+
+  const result = `${terminalUISupport.color ? chalk.red(tag) : tag} ${message}`;
+
+  if (log) {
+    logger.log(result);
+  } else {
+    return result;
+  }
+};
+
+const uiCommandDisabledBanner = (
+  command,
+  url = undefined,
+  message = undefined
+) => {
+  const i18nKey = 'lib.ui';
+
+  const tag =
+    message ||
+    i18n(`${i18nKey}.disabledMessage`, {
+      command,
+      url: url ? uiLink(i18n(`${i18nKey}.disabledUrlText`), url) : undefined,
+    });
+
+  logger.log();
+  uiLine();
+  uiErrorTag(tag, true);
+  uiLine();
+  logger.log();
+};
+
+const uiDeprecatedDescription = (message, command, url = undefined) => {
+  const i18nKey = 'lib.ui';
+
+  return i18n(`${i18nKey}.deprecatedDescription`, {
```

Comment:
If you wanted, this could probably also return `uiDeprecatedTag` similar to how uiDeprecatedMessage does it. Just to save a little on LOC when calling